14/solve_1.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+lines = File.readlines(ARGV[0]).map(&:strip)
```

Comment:
Reading input file from command line, but total time from hardwired constant is not systematic (yes, I am in Ondra mode - I screwed cooking meat for dinner, so I am mean to others now!)